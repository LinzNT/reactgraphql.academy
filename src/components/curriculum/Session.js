import React from 'react'
import { Li } from '../layout/Ul'

const Session = ({ hideTitle, children, title }) =>
  hideTitle ? (
    children
  ) : (
    <Li>
      {title}
      {children}
    </Li>
  )

export default Session
