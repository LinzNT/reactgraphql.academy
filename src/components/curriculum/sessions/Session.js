import React from 'react'
import { Li } from '../../layout/Ul'

const Session = ({ children, title }) =>
  !title ? (
    children
  ) : (
    <Li>
      {title}
      {children}
    </Li>
  )

export default Session
