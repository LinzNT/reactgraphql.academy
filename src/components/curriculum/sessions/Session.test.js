import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import Session from './Session'

describe('<Session />', () => {
  it('should render component', () => {
    const wrapper = shallow(<Session />)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
