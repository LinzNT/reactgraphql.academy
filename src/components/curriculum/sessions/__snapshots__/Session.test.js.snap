// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Session /> should render component 1`] = `""`;
